test/functional/rpc_setban.py

```diff
@@ -60,24 +60,6 @@ def run_test(self):
         assert not self.is_banned(node, tor_addr)
         assert not self.is_banned(node, ip_addr)
 
-        node.setban(tor_addr, "add")
-        assert self.is_banned(node, tor_addr)
-        assert not self.is_banned(node, ip_addr)
```

Comment:
Seems like this would also remove testing of banning an onion address, which doesn't seem ideal. Would recommend keeping this.

Reply:
Done.